### DISCUSSION TITLE
Key repetition is removed in Zed: v0.157.1 (Zed Preview) ?

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#18998] [Feature Requests]
Why?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
